src/modules/SX126x.h

```diff
@@ -604,6 +604,17 @@ class SX126x: public PhysicalLayer {
     */
     int16_t setSyncWord(uint8_t* syncWord, uint8_t len);
 
+    /*!
+      \brief Sets FSK sync word in the form of array of up to 8 bytes.
+
+      \param syncWord FSK sync word to be set.
+
+      \param len FSK sync word length in bits.
+
+      \returns \ref status_codes
+    */
+    int16_t setSyncBits(uint8_t *sync, uint8_t bitsLen);
```

Comment:
The name of the method should also be added to keywords.txt (please use true tab there), and it would be nice to have it in the SX126x_Settings example.

Reply:
Ok, I will try to add that. Now I'm just not sure how to "update" Pull request (GitHub newbie)..